torch/_torch_docs.py

```diff
@@ -2667,6 +2667,34 @@ def merge_dicts(*dicts):
 
 """.format(**common_args))
 
+add_docstr(torch.logical_and,
+           r"""
+logical_and(input, other, out=None) -> Tensor
+
+Computes the element-wise logical AND of the given input tensors. Zeros are treated as ``False`` and nonzeros are
+treated as ``True``.
+
+Args:
+    {input}
+    other (Tensor): the tensor to compute AND with
+    {out}
+
+Example::
+
+    >>> torch.logical_and(torch.tensor([True, False, True]), torch.tensor([True, False, False]))
+    tensor([ False, False,  True])
```

Comment:
Extra space chars make me think this example isn't pasted from a python prompt -- I think the last value should be `False` for `and` right? True and False for index `2`?

Reply:
This is probably my negligence in copying over the example from `logical_xor`. I've corrected this example now